prime-router/src/main/kotlin/tokens/SenderUtils.kt

```diff
@@ -0,0 +1,131 @@
+package gov.cdc.prime.router.tokens
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.nimbusds.jose.jwk.JWK
+import com.nimbusds.jose.jwk.KeyType
+import gov.cdc.prime.router.Sender
+import gov.cdc.prime.router.cli.SettingCommand
+import io.jsonwebtoken.Jwts
+import java.io.File
+import java.net.URL
+import java.security.PrivateKey
+import java.util.Date
+import java.util.UUID
+
+class SenderUtils {
+
+    companion object {
+        /**
+         * Generate a signed JWT, representing a request for authentication from a Sender, using a private key.
+         * This is done by the Sender, not by ReportStream. This method is here for testing, and as an example.
+         */
+        fun generateSenderToken(
+            sender: Sender,
+            baseUrl: String,
+            privateKey: PrivateKey,
+            keyId: String,
+            expirationSecondsFromNow: Int = 300,
+        ): String {
+            val jws = Jwts.builder()
+                .setHeaderParam("kid", keyId) // kid
+                .setHeaderParam("typ", "JWT") // typ
+                .setIssuer(sender.fullName) // iss
```

Comment:
As I read the spec, the `iss` should be CDC PRIME or some constant that represents us. 

> The "iss" (issuer) claim identifies the principal that issued the
   JWT.  The processing of this claim is generally application specific.
   The "iss" value is a case-sensitive string containing a StringOrURI
   value.  Use of this claim is OPTIONAL.

>